producer/policyauthorization.go

```diff
@@ -108,7 +108,7 @@ func handleMediaSubComponent(smPolicy *pcf_context.UeSmPolicyData, medComp *mode
 				var ul, dl bool
 				qosData, ul, dl = updateQosInMedSubComp(smPolicy.PolicyDecision.QosDecs[qosID], medComp, medSubComp)
 				if problemDetails := modifyRemainBitRate(smPolicy, &qosData, ul, dl); problemDetails != nil {
-					logger.PolicyAuthorizationlog.Errorf(problemDetails.Detail)
+					logger.PolicyAuthorizationlog.Errorln(problemDetails.Detail)
```

Comment:
Same as in the other comment, I am curious to know why this change?

Reply:
This fails in govet and aims to fix linting issues.